url75694980d5_fabioalmeid_tp01ftc_tgz-pJ8-agentes_jade_Centralizador_ExemploParserGramaticaCentralizadorJ8/src/monitor/Yylex.java

```diff
@@ -0,0 +1,446 @@
+		/* 12 */ YY_NO_ANCHOR,
+		/* 13 */ YY_NO_ANCHOR,
+		/* 14 */ YY_NO_ANCHOR,
+		/* 15 */ YY_NO_ANCHOR,
+		/* 16 */ YY_NO_ANCHOR,
+		/* 17 */ YY_NOT_ACCEPT,
+		/* 18 */ YY_NOT_ACCEPT,
+		/* 19 */ YY_NOT_ACCEPT,
+		/* 20 */ YY_NOT_ACCEPT,
+		/* 21 */ YY_NOT_ACCEPT,
+		/* 22 */ YY_NOT_ACCEPT,
+		/* 23 */ YY_NOT_ACCEPT,
+		/* 24 */ YY_NOT_ACCEPT,
+		/* 25 */ YY_NOT_ACCEPT,
+		/* 26 */ YY_NOT_ACCEPT,
+		/* 27 */ YY_NOT_ACCEPT,
+		/* 28 */ YY_NOT_ACCEPT,
+		/* 29 */ YY_NOT_ACCEPT,
+		/* 30 */ YY_NOT_ACCEPT,
+		/* 31 */ YY_NOT_ACCEPT,
+		/* 32 */ YY_NOT_ACCEPT,
+		/* 33 */ YY_NOT_ACCEPT,
+		/* 34 */ YY_NOT_ACCEPT,
+		/* 35 */ YY_NOT_ACCEPT,
+		/* 36 */ YY_NOT_ACCEPT,
+		/* 37 */ YY_NOT_ACCEPT,
+		/* 38 */ YY_NOT_ACCEPT,
+		/* 39 */ YY_NOT_ACCEPT,
+		/* 40 */ YY_NOT_ACCEPT,
+		/* 41 */ YY_NOT_ACCEPT,
+		/* 42 */ YY_NOT_ACCEPT,
+		/* 43 */ YY_NOT_ACCEPT,
+		/* 44 */ YY_NOT_ACCEPT,
+		/* 45 */ YY_NOT_ACCEPT,
+		/* 46 */ YY_NOT_ACCEPT,
+		/* 47 */ YY_NOT_ACCEPT,
+		/* 48 */ YY_NOT_ACCEPT,
+		/* 49 */ YY_NOT_ACCEPT,
+		/* 50 */ YY_NOT_ACCEPT,
+		/* 51 */ YY_NOT_ACCEPT,
+		/* 52 */ YY_NOT_ACCEPT,
+		/* 53 */ YY_NOT_ACCEPT,
+		/* 54 */ YY_NOT_ACCEPT,
+		/* 55 */ YY_NOT_ACCEPT,
+		/* 56 */ YY_NOT_ACCEPT,
+		/* 57 */ YY_NOT_ACCEPT,
+		/* 58 */ YY_NOT_ACCEPT,
+		/* 59 */ YY_NOT_ACCEPT,
+		/* 60 */ YY_NOT_ACCEPT,
+		/* 61 */ YY_NOT_ACCEPT,
+		/* 62 */ YY_NOT_ACCEPT,
+		/* 63 */ YY_NOT_ACCEPT,
+		/* 64 */ YY_NOT_ACCEPT,
+		/* 65 */ YY_NOT_ACCEPT,
+		/* 66 */ YY_NOT_ACCEPT,
+		/* 67 */ YY_NOT_ACCEPT,
+		/* 68 */ YY_NOT_ACCEPT,
+		/* 69 */ YY_NOT_ACCEPT,
+		/* 70 */ YY_NOT_ACCEPT,
+		/* 71 */ YY_NOT_ACCEPT,
+		/* 72 */ YY_NOT_ACCEPT,
+		/* 73 */ YY_NOT_ACCEPT,
+		/* 74 */ YY_NOT_ACCEPT,
+		/* 75 */ YY_NOT_ACCEPT
+	};
+	private int yy_cmap[] = unpackFromString(1,65538,
+"0:9,28:2,0,28:2,0:18,9,0:14,12,27:10,20,0:6,17,22,25,0:4,26,0:3,13,0:3,14,0" +
+":3,1,0:12,6,23,0,10,2,0,11,21,18,0:2,19,3,24,16,4,0,5,15,7,8,0:65418,29:2")[0];
+
+	private int yy_rmap[] = unpackFromString(1,76,
+"0,1,2,1:3,3,1:10,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,2" +
+"5,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,5" +
+"0,51,52,53,54,55,56,57,58,59,60,61,62")[0];
+
+	private int yy_nxt[][] = unpackFromString(63,30,
+"-1,17,1,2,-1:2,18,-1:2,3,-1,19,-1:2,20,-1:5,4,21,22,-1:2,5,72,6,3,7,-1:33,2" +
+"4,-1:7,73,-1:45,6,-1:4,23,-1:42,8,-1:26,25,-1:22,26,-1:44,9,-1:27,27,-1:14," +
+"28,-1:52,29,-1:13,30,-1:21,31,-1:46,32,-1:14,34,-1:36,10,-1:31,11,-1:31,63," +
+"-1:32,36,-1:27,37,-1:15,64,-1:40,12,-1:21,68,-1:35,39,-1:24,40,-1:42,66,-1:" +
+"26,43,-1:21,44,-1:28,45,-1:31,46,-1:43,47,-1:14,69,-1:38,70,-1:30,48,-1:35," +
+"51,-1:11,53,-1:30,54,-1:28,13,-1:29,14,-1:32,55,-1:22,71,-1:37,56,-1:21,15," +
+"-1:45,58,-1:17,59,-1:42,16,-1:39,7,-1:3,33,-1:36,35,-1:34,38,-1:19,65,-1:30" +
+",42,-1:39,75,-1:37,52,-1:10,41,-1:29,49,-1:29,50,-1:29,57,-1:26,61,-1:39,62" +
+",-1:35,67,-1:34,74,-1:6");
+
+	public java_cup.runtime.Symbol next_token ()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This method contains 99 lines of code, not including blank lines or lines with only comments, Java punctuation characters, identifiers, or literals. By comparison, 99% of the methods in the CodeGuru reference dataset contain fewer lines of code. Large methods might be difficult to read and have logic that is hard to understand and test. We recommend that you simplify this method or break it into multiple methods.